### ISSUE TITLE
more comments？

### ISSUE THREAD

**Issue by tomatoccoo** [GoodAI/badger-2019#2]
it is hard to read the codes, hope for more comments ^_^

**Comment by dan9thsense** [comment]
I agree with this comment.  I read the paper from dec 2019 and would like to try the badger architecture, but need an intro tutorial or some framework to get started.
